diff --git a/src/gui/widgets/mpv/mpvwidget.cpp b/src/gui/widgets/mpv/mpvwidget.cpp
index a2c1f7f..c804551 100644
--- a/src/gui/widgets/mpv/mpvwidget.cpp
+++ b/src/gui/widgets/mpv/mpvwidget.cpp
@@ -834,7 +834,9 @@ void MpvWidget::preventScreenDimming()
     );
 #elif defined(Q_OS_UNIX) && !defined(Q_OS_DARWIN)
     QDBusInterface screenSaver(
-        "org.freedesktop.ScreenSaver", "/org/freedesktop/ScreenSaver"
+        "org.freedesktop.ScreenSaver",
+        "/ScreenSaver",
+        "org.freedesktop.ScreenSaver"
     );
     QDBusMessage reply = screenSaver.call(
         "Inhibit", "io.github.ripose-jp.memento", "Playing a video"
